Source/Scene/BatchTexture.js

```diff
@@ -18,7 +18,8 @@ import Texture from "../Renderer/Texture.js";
  *
  * @param {Object} options Object with the following properties:
  * @param {Number} featuresLength The number of features in the batch table or feature table
- * @param {Cesium3DTileContent} content The content this batch texture belongs to. Used for creating the picking texture.
+ * @param {Cesium3DTileContent|ModelFeatureTable} content The content this batch texture belongs to. Used for creating the picking texture.
```

Comment:
Update the description to explain how this gets used when it's a ModelFeatureTable